analyzers/rspec/cs/S5753_c#.html

```diff
@@ -0,0 +1,74 @@
+<p>ASP.NET 1.1+ comes with a feature called <em>Request Validation</em>, preventing the server to accept content containing un-encoded HTML. This
+feature comes as a first protection layer against Cross-Site Scripting (XSS) attacks and act as a simple Web Application Firewall (WAF) rejecting
+requests potentially containing malicious content.</p>
+<p>While this feature is not a silver bullet to prevent all XSS attacks, it helps to catch basic ones. It will for example prevent <code>&lt;script
+type="text/javascript" src="https://malicious.domain/payload.js"&gt;</code> to reach your Controller.</p>
+<p>Note: <em>Request Validation</em> feature being only available for ASP.NET, no Security Hotspot is raised on ASP.NET Core applications.</p>
+<h2>Ask Yourself Whether</h2>
+<ul>
+  <li> the developer doesn't know the impact to deactivate the Request Validation feature </li>
+  <li> the web application accepts user supplied data </li>
+  <li> all user supplied data are not validated </li>
+</ul>
+<p>There is a risk if you answered yes to any of those questions.</p>
+<h2>Recommended Secure Coding Practices</h2>
+<ul>
+  <li> Activate the Request Validation feature for all HTTP requests </li>
+</ul>
+<h2>Sensitive Code Example</h2>
+<p>At Controller level:</p>
+<pre>
+[ValidateInput(false)]
+public ActionResult Welcome(string name)
+{
+  ...
+}
+</pre>
+<p>At application level, configured in the Web.config file:</p>
+<pre>
+&lt;configuration&gt;
```

Comment:
Just raising awareness, that this `web.config` example is present here. Little ahead of it's time.

Reply:
indeed. as this won't close the issue, I find this tolerable. If we don't solve the other bit, we'll update the RSPEC and let AppSec know.

Reply:
I rebased it on a feature branch